### PR TITLE
wxWidgets: update to 3.1.4.

### PR DESCRIPTION
**Pull request by sirikid**
I think wxWidgets is mistaking Musl for Windows. @Hoshpak have you seen this error?

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -4040,3 +4040,17 @@ libndr-samba4.so samba-libs-4.13.2_1
 libsmb-transport-samba4.so samba-libs-4.13.2_1
 libutil-cmdline-samba4.so samba-libs-4.13.2_1
 libwinbind-client-samba4.so samba-libs-4.13.2_1
+libwx_baseu-3.1.so.4 wxWidgets-common-3.1.4_1
```
Comment (ericonr) [inline common/shlibs:4043]: You should update the old wx entries, not add new ones.

File: srcpkgs/wxWidgets/template [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
 pkgname=wxWidgets
-version=3.0.5.1
+version=3.1.4
 revision=1
-configure_args="--enable-unicode --with-opengl --with-sdl --with-libmspack
- --with-libnotify --enable-mediactrl --with-gtk=2 --disable-webview"
 build_style=gnu-configure
+configure_args="--disable-option-checking --disable-webview
+ --enable-mediactrl --enable-unicode
+ --with-gtk=2 --with-libmspack --with-libnotify --with-opengl --with-sdl"
```
Comment (ericonr) [inline srcpkgs/wxWidgets/template:8]: Is it possible to switch to gtk3 here?

**Review by ericonr — COMMENTED** [review]
Is this working for wxpython? Last I tried, it didn't really work. Both wxpython and wxpython4 likely require a revbump.

File: common/shlibs (hunk 1 above)
Comment (sirikid) [reply common/shlibs:4043]: Thanks. I wanted to ask about this.

File: srcpkgs/wxWidgets/template (hunk 2 above)
Comment (sirikid) [reply srcpkgs/wxWidgets/template:8]: There is another package for Gtk3.

Comment (sirikid) [reply srcpkgs/wxWidgets/template:8]: It seems most of the packages can be recompiled for wxGtk3, I will try to do this in a separate branch.

**Comment by Johnnynator** [comment]
> Please notice that while 3.1.4 is officially a “development” version because it is not fully compatible with the “stable” 3.0.x, the list of backwards incompatible changes is very short, so you shouldn’t have any problems updating to this version from 3.0.x in practice, and you’re encouraged to use this release, including in production.

We should stay with stable.

**Comment by sirikid** [comment]
> > Please notice that while 3.1.4 is officially a “development” version because it is not fully compatible with the “stable” 3.0.x, the list of backwards incompatible changes is very short, so you shouldn’t have any problems updating to this version from 3.0.x in practice, and you’re encouraged to use this release, including in production.
> 
> We should stay with stable.

Understood, I will prepare PR anyway.

**Pull request closed without merging**